### PR TITLE
Feature/gui

### PR DESCRIPTION
**Pull request by Ishadijcks**
Does not work on my machine, but does on @RvdK. Can everyone please test if it works on their machine? If it works for everyone it might be just me, and can be merged

Transitions between activities
Persistent settings menu.
Consistent toolbar across applications

### CODE REVIEW

**Comment by RvdK** [comment]
Crashes are now fixed the issue was android 6

File: BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/SettingsActivity.java [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                int index = listPreference.findIndexOfValue(stringValue);
+
+                // Set the summary to reflect the new value.
+                preference.setSummary(
+                        index >= 0
+                                ? listPreference.getEntries()[index]
+                                : null);
+
+            } else if (preference instanceof RingtonePreference) {
+                // For ringtone preferences, look up the correct display value
```
Comment (RvdK) [inline BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/SettingsActivity.java]: what do we use this for

File: BankChain/app/src/main/res/raw/environment_example.xml [hunk 2]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (RvdK) [inline BankChain/app/src/main/res/raw/environment_example.xml]: why throw this away?

Comment (joskuijpers) [reply BankChain/app/src/main/res/raw/environment_example.xml]: Yes you must not throw this away

File: BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/SettingsActivity.java (hunk 1 above)
Comment (Ishadijcks) [reply BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/activities/SettingsActivity.java]: We don't, it was autocreated by android studio. I have removed it.

File: BankChain/app/src/main/res/raw/environment_example.xml (hunk 2 above)
Comment (Ishadijcks) [reply BankChain/app/src/main/res/raw/environment_example.xml]: An accident, I've added it again.

**Comment by Ishadijcks** [comment]
I have fixed all review comments.

**Review by RvdK — APPROVED** [review]

**Pull request merged**